tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Features/Login with a valid client 3540.feature

```diff
@@ -10,6 +10,22 @@ Background:
 
 @3540
 Scenario: 1. Logging in with an existing client with valid credentials
-    When the user navigates to the login page with return url http://localhost:8072
+    When the user navigates to a restricted web page
+    Then the user is redirected to identity server page account/login
```

Comment:
I would be tempted to reword this to:
Then the user is redirected to the login page. 

My thinking here is does the user really need to know its an Identity server? In future we may need to distinguish between the Buying Catagloue login page and the NHS Email login page.

Reply:
hmm, I initially added the identity server vs client parameter to the redirected page, so that we had one generic step for redirections that could deal with the differing ports for identity vs client. I see what you mean though, it being an identity server is somewhat irrelevant. Maybe I could change it from identity server to... authentication page, or something like that, as the parameter passed in?

Reply:
Yeah I don't think the user needs to know whether it is a client or a server. All they really know is a web page. Yeah definitely rename it. I feel like login page would be better, authentication sounds more technical to a general user.